client/clientservice/CMakeLists.txt

```diff
@@ -1,13 +1,13 @@
 find_package(GRPC REQUIRED)
 find_package(Boost ${MIN_BOOST_VERSION} COMPONENTS program_options REQUIRED)
 find_package(jaegertracing REQUIRED)
```

Comment:
Shouldn't there be a version here as well?

Reply:
Not dealing with it right now, my intention is to remove jaeger from the library